internal/credentials/credentials.go

```diff
@@ -196,6 +210,7 @@ func (cs *CredentialsService) Delete(guid string) error {
 
 	_, exists := table[guid]
 	if !exists {
+		cs.log.Debug("Credential does not exist", "credential", guid)
```

Comment:
Further not following why you've got logging messages in here if you are initially going to a discarding logger which won't do anything?

Reply:
We are using proper loggers within the endpoints that create `NewCredentialService`, the only no-op logger is for `cli` tooling